src/decorators/legacy/index.mjs

```diff
@@ -0,0 +1,32 @@
+import Pkg from './index.js';
```

Comment:
Don't forget to export the `Associate` ones as well

Reply:
Done